doc/admin-guide/plugins/wasm.en.rst

```diff
@@ -97,14 +97,15 @@ generated wasm modules with the plugin.
 Runtime can be chosen by changing the ``runtime`` field inside the yaml configuration file for the plugin.
 ``ats.wasm.runtime.wamr`` is for WAMR while ``ats.wasm.runtime.wasmedge`` is for WasmEdge.
 
+The plugin can also take more than one yaml file as arguments and can thus load more than one wasm modules.
+
```

Comment:
Can we also add an example of a plugin config definition?

Reply:
The example plugin config can be found in the examples directory in the - https://github.com/apache/trafficserver/tree/master/plugins/experimental/wasm/examples